probes/pom.xml

```diff
@@ -16,6 +16,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>commons-codec</groupId>
```

Comment:
Good catch, thanks!

I don't now if this is the best solution, but the `sun.misc.Base64Decoder` is on the checkstyle blacklist. So I had a little search what to use instead. I'll add the JAR to the provisioner asap.